Formula/go.rb

```diff
@@ -34,38 +50,38 @@ class Go < Formula
     sha256 "5f570b6c7aa2d7caa6c715af6dce6fa30d7fbd5acc46fac8fbc3232270956f9e" => :mavericks
   end
 
-  devel do
-    url "https://storage.googleapis.com/golang/go1.7rc5.src.tar.gz"
-    version "1.7rc5"
-    sha256 "206c90e797e66335fe134052568f63a493f27b86f765087add390d5fb4c596c4"
-
-    # Should use the last stable binary release to bootstrap.
-    # Not the case here because 1.6.3 is lacking a fix for an issue which breaks
-    # compile on macOS Sierra; in future this should share bootstrap with stable.
-    resource "gobootstrap" do
-      url "https://storage.googleapis.com/golang/go1.7rc5.darwin-amd64.tar.gz"
+  if MacOS.version < :sierra
```

Comment:
Although I guess in this case it shouldn't exist without so actually: this is fine.